block/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
+yarn.lock
```

Comment:
@buchansm my understanding is that we want the .lock file under version control so that each of our checkouts use the same dependency versions for testing consistency. 

Let me know if you disagree or if there's an issue with the lock file.

Reply:
Or if this is just to enforce npm over yarn that's alright with me.